addons/mail/static/src/components/message_list/message_list.js

```diff
@@ -443,20 +447,32 @@ class MessageList extends Component {
             return false;
         }
         const loadMoreRect = loadMore.getBoundingClientRect();
-        const elRect = this.el.getBoundingClientRect();
+        const elRect = this.scrollableElement.getBoundingClientRect();
         const isInvisible = loadMoreRect.top > elRect.bottom || loadMoreRect.bottom < elRect.top;
         return !isInvisible;
     }
 
     /**
      * @private
      */
-    _loadMore() {
+    async _loadMore() {
```

Comment:
Same here and everywhere else, don't introduce more async code

Reply:
@seb-odoo actually `loadMoreMessages` is an async method. So I have only handled `_loadMore` method as an async and only where that is required in throttled function.

Reply:
But why the change? It was working fine before, we don't want to introduce an async point in that method if we can avoid it.